storage_broker/src/bin/storage_broker.rs

```diff
@@ -424,12 +424,16 @@ async fn http1_handler(
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    // initialize sentry if SENTRY_DSN is provided
-    let _sentry_guard = init_sentry(Some(GIT_VERSION.into()), &[]);
-
     let args = Args::parse();
 
+    // important to keep the order of:
+    // 1. init logging
+    // 2. tracing panic hook
+    // 3. sentry
     logging::init(LogFormat::from_config(&args.log_format)?)?;
+    logging::replace_panic_hook_with_tracing_panic_hook().disarm();
```

Comment:
Maybe it's just me, `replace().disarm()` looks a bit confusing, as if we disabled the thing we had just installed.

Reply:
Maybe `forget` or plain using `std::mem::forget` on the value. But I also added `#[must_use]` so there'd be a warning if you'd remove the disarm while wondering what it is.

Reply:
Yeah, `forget` sounds better!

Reply:
[d2964f9](https://github.com/neondatabase/neon/pull/3634/commits/d2964f990e2a4f00789aae2d73b581191e09b724)